

print(false;

with(x);   // not an error